apps/browser/qml/pages/components/FavoriteGrid.qml

```diff
@@ -55,11 +55,13 @@ IconGridViewBase {
 
         onAddToLauncher: {
             // url, title, favicon
-            pageStack.animatorPush(addToLauncherDialog,
+            pageStack.animatorPush("AddToAppGridDialog.qml",
                                    {
                                        "url": url,
                                        "title": title,
-                                       "icon": favicon
+                                       "icon": favicon,
+                                       "desktopBookmarkWriter": desktopBookmarkWriter,
+                                       "bookmarkWriterParent": pageStack
```

Comment:
here the favorite grid was just fine... as lifetime to of this was the same as the pagestack's. But yes, pageStack works here as well.